st2common/bin/BUILD

```diff
@@ -1 +1,9 @@
-python_sources()
+python_sources(
+    sources=["*.py", "st2*", "!st2ctl", "!st2-self-check", "!st2-run-pack-tests"],
+)
+
+shell_sources(
+    name="shell",
+    sources=["st2ctl", "st2-self-check", "st2-run-pack-tests"],
```

Comment:
Notice how I've got the `st2*` glob on `python_sources`.
That would match the files from `shell_sources`, so we exclude them from matching using `!`.